snippets/kubernetes.json

```diff
@@ -460,5 +460,834 @@
+    },
+    "Ingress TLS section": {
+        "body": [
+            "tls:",
+            "- secretName: ${2:$1.tls}",
+            "  hosts:",
+            "  - ${1:host}"
+        ],
+        "prefix": "ingtls"
+    },
+    "cert-manager certificate": {
+        "body": [
+            "apiVersion: certmanager.k8s.io/v1alpha1",
+            "kind: Certificate",
+            "metadata:",
+            "  name: ${1:name}",
+            "  namespace: ${2:namespace}",
+            "spec:",
+            "  secretName: ${4:$3.tls}",
+            "  dnsNames:",
+            "  - ${3:some.domain.com}",
+            "  acme:",
+            "    config:",
+            "    - dns01:",
+            "        provider: ${4:prod}",
+            "      domains: [ $3 ]",
+            "  issuerRef:",
+            "    name: ${5:letsencrypt}",
+            "    kind: ClusterIssuer"
+        ],
+        "prefix": "cert"
+    },
+    "Ingress": {
+        "body": [
+            "apiVersion: networking.k8s.io/v1",
+            "kind: Ingress",
+            "metadata:",
+            "  annotations:",
+            "    nginx.ingress.kubernetes.io/rewrite-target: /",
+            "    kubernetes.io/ingress.class: nginx",
+            "    cert-manager.io/cluster-issuer: \"letsencrypt-prod\"",
+            "  name: ${1:name}",
+            "  namespace: ${2:default}",
+            "spec:",
+            "  ${5:tls:",
+            "  - secretName: ${4:$3.tls}",
+            "    hosts:",
+            "    - $3",
+            "  }rules:",
+            "  - host: ${3:host.tld}",
+            "    http:",
+            "      paths:",
+            "      - path: ${7:/}",
+            "        pathType: Prefix",
+            "        backend:",
+            "          service:",
+            "            name: ${8:service}",
+            "            port:",
+            "              number: ${9:portNumberOrName}"
+        ],
+        "prefix": "ing"
+    },
+    "Resources": {
+        "body": [
+            "resources:",
+            "  requests:",
+            "    cpu: ${1:100m}",
+            "    memory: ${2:200Mi}",
+            "  ${5:limits:",
+            "    cpu: ${3:$1}",
+            "    memory: ${4:$2}}$0"
+        ],
+        "prefix": "res"
+    },
+    "Namespace": {
+        "body": [
+            "apiVersion: v1",
+            "kind: Namespace",
+            "metadata:",
+            "  name: ${1:name}"
+        ],
+        "prefix": "ns"
+    },
+    "Deployment Strategy": {
+        "body": [
+            "strategy:",
+            "  type: ${1:RollingUpdate|Recreate}",
+            "  rollingUpdate:",
+            "    maxSurge: ${2:1}",
+            "    maxUnavailable: ${3:1}$0"
+        ],
+        "prefix": "strat"
+    },
+    "external dns": {
+        "body": [
+            "annotations:",
+            "  external-dns.alpha.kubernetes.io/hostname: ${1:myname.mydomain.com}"
+        ],
+        "prefix": "edns"
+    },
+    "Kubernetes Cronjob": {
```

Comment:
Like on the other snippet, this cron snippet seems like it should be an update to the `k-cronjob` snippet -- curious for your reasoning on adding this one instead of modifying the other. 

For context: 
![image](https://user-images.githubusercontent.com/56930510/176343398-e74df206-2242-46ea-8e3a-adf495d9eba3.png)